dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/messages/messages.properties

```diff
@@ -72,6 +72,6 @@ DeleteProjectError=An error occurred while trying to delete the contents of the
 
 ProjectSettingsUpdateErrorTitle=Project settings update error
 
-DockerTypeDisplayName=Other (Basic Container)
+DockerTypeDisplayName=Other (Codewind Basic Container)
```

Comment:
This label was decided on by design and ID.  We should check with them before changing it.

Reply:
This was requested to be changed by @tetchel to be consistent with VS code

Reply:
indeed https://github.com/eclipse/codewind-vscode/pull/224#issue-324718610